src/internal/nft-api.ts

```diff
@@ -378,6 +379,19 @@ export async function computeRarity(
   return getNftRarityFromRaw(response);
 }
 
+export async function summarizeNftAttributes(
+  config: AlchemyConfig,
+  contractAddress: string,
+  srcMethod = 'summarizeNFTAttributes'
+): Promise<NftAttributesSummary> {
```

Comment:
please rename `NftAttributesSummary` to `NftAttributesResponse`.

Reply:
Oky